winterfell/webapp/src/js/formRenderingService.js

```diff
@@ -27,3 +27,29 @@ app.factory('formRenderingService', ['$window', '$http', function ($window, $htt
         }
     }
 }]);
+
+/**
+ * Service to retrieve the formly template (which is just a static JSON file)
+ * for a given form.
+ */
+app.factory('formTemplateService', ['$http', function ($http) {
```

Comment:
There is a service called net.js which handles API endpoints to our server.
You should use that if you want to make HTTP requests